docs/pages/access-controls/device-trust/jamf-integration.mdx

```diff
@@ -0,0 +1,303 @@
+[auto-enrollment](#optional-enable-auto-enrollment) section.
+
+See the [Device Trust guide](./guide.mdx) for fundamental Device Trust concepts
+and behavior.
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
+
+(!docs/pages/includes/device-trust/prereqs.mdx!)
+
+## Step 1/4. Create a Jamf user
+
+Create a readonly Jamf user for inventory sync.
+
+1. Access https://yourtenant.jamfcloud.com/accounts.html, replacing "yourtenant"
+   with your Jamf Pro account.
+
+2. Create a new Standard Account with the following settings:
+
+   - Username: teleport (change as desired)
+   - Access Level: Full Access
+   - Privilege Set: Custom
+   - Access Status: Enabled
+   - Password: (a strong password of your choice)
+   - Privileges:
+     - Advanced Computer Searches: Read
+     - Computers: Read
+
+   Take note of the user and password created here for the next step.
+
+   User account setup: ![Jamf user setup](
+   ../../../img/access-controls/device-trust/jamf-setup-1.png)
+
+   Privileges setup: ![Jamf privileges setup](
+   ../../../img/access-controls/device-trust/jamf-setup-2.png)
+
+## Step 2/4. Configure Jamf service
+
+Jamf inventory sync is performed by a separate `teleport` process, configured
+using the `jamf_service` key. It is recommended to run the service isolated from
+other Teleport processes, as it requires the Jamf credentials created in the
+step above.
+
+Save the following file as `/var/lib/teleport.yaml` and edit as needed:
+
+```yaml
+version: v3
+teleport:
+  # Necessary to write devices back to Teleport.
+  proxy_server: teleport.example.com:443 # CHANGEME
+
+jamf_service:
+  enabled: true
+  name: jamf
+  api_endpoint: https://yourtenant.jamfcloud.com # CHANGEME
+  username: teleport                             # CHANGEME
+  password_file: /var/lib/teleport/jamf_password.txt
+
+auth_service:
+  enabled: false
+
+proxy_service:
+  enabled: false
+
+ssh_service:
+  enabled: false
+```
+
+Change the following settings, as appropriate:
+
+- teleport.proxy_server
+- jamf_service.api_endpoint
+- jamf_service.username
+
+Finally, write your Jamf password to the `/var/lib/teleport/jamf_password` file:
+
+```code
+$ nano /var/lib/teleport/jamf_password # or use your favorite editor
+
+# Only the OS user that runs `teleport` should have access to the password file.
+$ chmod 400 /var/lib/teleport/jamf_password
+$ chown teleport /var/lib/teleport/jamf_password
+```
+
+## Step 3/4. Create a join token
+
+Jamf service requires an MDM token to write devices to Teleport.
+Create the token as shown below:
+
+```code
+$ tctl tokens add --type=mdm
+The invite token: 07c0565aa280595d7645b2964c36ebd1
+This token will expire in 30 minutes.
+
+Use this token to add an MDM service to Teleport.
+
+> teleport start \
+   --token=07c0565aa280595d7645b2964c36ebd1 \
+   --ca-pin=sha256:4b32d9c54b2b3332019d5f0720b8f9a603de03ace07d308bcd743465eee1f200 \
+   -c=/path/to/teleport.yaml
```

Comment:
```suggestion
   --config=/path/to/teleport.yaml
```

I think it's either got to be `-c path` or `--config=path`.

In any case, I always prefer the long-form flag for written docs, and leave the short flags for power users.

Reply:
This one is the output from "tctl tokens add", verbatim. I can touch it up here, but it seems wrong without touching the .go first. Let me do that.

Reply:
https://github.com/gravitational/teleport/pull/28232